web_widget_float_formula/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright GRAP
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Web Widget - Formulas in Float Fields',
+    'summary': 'Allow use of simple formulas in float fields',
+    'version': '11.0.1.0.0',
+    'category': 'Web',
+    'author': 'GRAP, LasLabs, Odoo Community Association (OCA)',
+    'website': 'http://www.grap.coop',
```

Comment:
@hgminh95 URL must be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)